footprints/templates/clientside/footprint_book.html

```diff
@@ -319,6 +319,7 @@
                         </div>
                     <% } else { %>
                         <%=imprint.standardized_identifier[i].identifier%>
+                        <p class="text-muted">The Bibliography of the Hebrew Book is currently in transition, so links may be out of date. Please be patient while we wait for this transition to be completed.</p>
```

Comment:
@ndittren , I thought we only add this paragraph if the authority is `Bibliography of the Hebrew Book`.  So it shouldn't be under the `else` condition?

Reply:
@ndittren Let's get rid of the `else if` on line 320, and move line 321-322 within the above `else if` block.  Tha way we don't have to do another `else if` check just for rendering that paragraph.

Reply:
@dc3788 If we move that up then the `else` block on line 323 executes and we get two identifier numbers printed. We can move the paragraph up without the identifier number but then it will be above the number instead of below